app/main/posts/collections/mode-context.html

```diff
@@ -112,7 +112,7 @@ <h1 class="mode-context-title">
         <!--// END IF //-->
         <div class="form-field">
             <!-- mode context bar is only available in the map so it should take you back there -->
-            <a ui-sref="posts.map.all" type="button" class="button-link">
+            <a ng-click="clearFilters()" type="button" class="button-link">
```

Comment:
FYI because I'm not 100% on what expected here . . but this will clear any filters that the user set while being on a saved search or collection, which is not what happens usually when they navigate away from data to map, map to data, etc. Is that OK?  -I expect it would be OK but, not sure-

Reply:
Yes. Thats expected. The button says 'return to all posts' ... its not really 'all posts' unless we clear filters.